exla/mix.exs

```diff
@@ -33,7 +33,9 @@ defmodule EXLA.MixProject do
   defp deps do
     [
       {:nx, path: "../nx"},
-      {:elixir_make, "~> 0.6"},
+      # TODO: adjust
+      {:xla, "~> 0.1.1-dev", runtime: false, github: "jonatanklosko/xla", branch: "jk-build"},
```

Comment:
Note: once the build succeeds we can update this reference and should be ready to go :)